Resources/Prototypes/_Lust/Entities/Customization/Markings/demon.yml

```
@@ -0,0 +1,35 @@
+- type: marking
+  id: DemonCurvedHorns
+  bodyPart: HeadTop
+  markingCategory: HeadTop
+  speciesRestriction: [Demon]
+  sprites:
+  - sprite: _Lust/Mobs/Species/Demon/markings.rsi
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)